docs/SERVER.md

```diff
@@ -0,0 +1,311 @@
+
+0. Firtsly, install `smp-server`:
+
+   - Manual deployment:
+     - [Compiling from source](https://github.com/simplex-chat/simplexmq#using-your-distribution)
+     - [Using pre-compiled binaries](https://github.com/simplex-chat/simplexmq#install-binaries)
+
+   - Alternatively, you can deploy `smp-server` using:
+     - [Docker container](https://github.com/simplex-chat/simplexmq#using-docker-1)
+     - [Linode StackScript](https://github.com/simplex-chat/simplexmq#deploy-smp-server-on-linode)
+
+Manual installation require some preparations:
+
+1. Create user and group for `smp-server`:
+
+   ```sh
+   sudo useradd -m smp
+   ```
+
+2. Create necessary directories and assign permissions:
+
+   ```sh
+   sudo mkdir -p /var/opt/simplex /etc/opt/simplex
+   sudo chown smp:smp /var/opt/simplex /etc/opt/simplex
+   ```
+
+3. Allow `smp-server` port in firewall:
+
+   ```sh
+   # For Ubuntu
+   ufw allow 5233
+   ```
+
+4. **Optional** — If you're using distribution with `systemd`, create `/etc/systemd/system/smp-server.service` file with the following content:
+
+   ```sh
+   [Unit]
+   Description=SMP server
+   [Service]
+   User=smp
+   Group=smp
+   Type=simple
+   ExecStart=smp-server start
+   ExecStopPost=/usr/bin/env sh -c '[ -e "/var/opt/simplex/smp-server-store.log" ] && cp "/var/opt/simplex/smp-server-store.log" "/var/opt/simplex/smp-server-store.log.bak"'
+   KillSignal=SIGINT
+   TimeoutStopSec=infinity
+   Restart=always
+   RestartSec=10
+   LimitNOFILE=65535
+   [Install]
+   WantedBy=multi-user.target
+   ```
+
+   And execute `sudo systemctl daemon-reload`.
+
+## Configuration
+
+To investigate which options are available, execute `smp-server` without flags:
+
+```sh
+sudo su smp -c smp-server
+
+...
+Available commands:
+  init                     Initialize server - creates /etc/opt/simplex and
+                           /var/opt/simplex directories and configuration files
+  start                    Start server (configuration:
+                           /etc/opt/simplex/smp-server.ini)
+  delete                   Delete configuration and log files
+```
+
+You can get further help by executing `sudo su smp -c "smp-server <command> -h"`
+
+After that, we need to configure `smp-server`:
+
+### Interactively
+
+Execute the following command:
+
+```sh
+sudo su smp -c "smp-server init"
+```
+
+There will be several options to consider:
+
+- `Enable store log to restore queues and messages on server restart (Yn):`
+
+  Enter `y` to enable saving and restoring messages upon restarting the server.
+  
+- `Enable logging daily statistics (yN):`
+
+  Enter `y` to enable logging statistics for `Prometheus/Grafana`.
+  
+- `Require a password to create new messaging queues?`
+
+  Enter `r` or your arbitrary password to password-protect `smp-server`, or `n` to disable password protection.
+  
+- `Enter server FQDN or IP address for certificate (127.0.0.1):`
+
+  Enter your domain or ip address to serve `smp-server` from.
```

Comment:
```suggestion
Enter your domain or ip address that your smp-server is running on - it will be included in server certificates and also printed as part of server address.
```